client/js/store/map/ProcedureMapSettings.js

```diff
@@ -39,9 +39,10 @@ export default {
         .then(response => checkResponse(response))
     },
 
-    fetchProcedureMapSettings ({ commit }, procedureId) {
+    fetchProcedureMapSettings ({ commit }, args= { procedureId, isMaster }) {
```

Comment:
Should we give `isMaster` a default value of `false`? I know that it will also work if it is `undefined`, but it seems cleaner to define a default value.

Reply:
Probably better, I added it here: [58ef831](https://github.com/demos-europe/demosplan-core/pull/3738/commits/58ef83174e231523c97a87673294f5961a2d34c8)

Reply:
Does it work like that? I think it has to be `isMaster = false`

Reply:
`isMaster = false` throws an error with `args`. But somehow now it works as expected without the args. Done here [ecf51f2](https://github.com/demos-europe/demosplan-core/pull/3738/commits/ecf51f259e7f05b7e00021b5d6accb312866f014)